.github/workflows/tests.yml

```diff
@@ -50,6 +50,9 @@ jobs:
           - ":presto-spark-base -P presto-spark-tests-all-queries"
           - ":presto-spark-base -P presto-spark-tests-spill-queries"
     timeout-minutes: 80
+    concurrency:
+      group: ${{ github.workflow }}-test-${{ matrix.modules }}-${{ github.event.pull_request.number }}
```

Comment:
This is notable because each matrix item effectively creates another job that is mostly equivalent, so one needs to distinguish by the matrix instance.

I'm not sure if github cares about particular characters in `concurrency.group`. I'm assuming it doesn't.